src/layout/BlockTime.tsx

```diff
@@ -19,6 +19,13 @@ const styles = StyleSheet.create({
 
 export function BlockTime({blockNumber}: Props) {
   const {timeStringParts} = useBlockTime(blockNumber);
```

Comment:
should we destruct it to make it more readable?
```suggestion
  const {timeStringParts} = useBlockTime(blockNumber);
  const [days, hours, minutes, seconds] = timeStringParts;
```